orcid-core/src/main/resources/i18n/api_en.properties

```diff
@@ -61,7 +61,7 @@ apiError.9018.userMessage=The ORCID record is locked.
 apiError.9018.moreInfo=http://members.orcid.org/api/api-error-codes
 apiError.9019.userMessage=The put code in the URL did not match that in the body of the message.
 apiError.9019.moreInfo=http://members.orcid.org/api/api-error-codes
-apiError.9020.userMessage=The put code in the URL did not match that in the body of the message.
+apiError.9020.userMessage=Invalid incoming message.
```

Comment:
@rcpeters A question. For the other issue regarding 2.0 Work validation, I created a couple of custom exceptions and associated them with 2 new error messages. Would you have rather preferred using the existing exception with a new messg? I need to fix another issue somewhat similar and I am in a dilemma now.